### ISSUE TITLE
Ceiling with Vlookup

### ISSUE THREAD

**Issue by colbybhearn** [EPPlusSoftware/EPPlus#1022]
Consider a formula where the `VLookup` function to select the value of the _significance_ parameter to the `Ceiling` function. When the `Vlookup` returns blank, Excel produces a 0 but EPPlus produces `#Value!` for the overall formula result.

I'll submit a PR to create a test for this.

Here's Excel versus EPPlus output (respectively)
[ceil-vlookup.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/12401822/ceil-vlookup.xlsx)
![image](https://github.com/EPPlusSoftware/EPPlus/assets/2199699/a4395c98-bad7-413d-8d14-357a957bcfd5)

![image](https://github.com/EPPlusSoftware/EPPlus/assets/2199699/30cdec08-be85-48bd-8eda-b627eb2beca0)

**Comment by colbybhearn** [comment]
Here's the sandbox code for that example:
```
 public static void ceilVlookup()
 {
     string filepath = @"ceil-vlookup.xlsx";
     ExcelPackage.LicenseContext = LicenseContext.Commercial;
     using (var package = new ExcelPackage(new FileInfo(filepath)))
     {
             package.Workbook.Calculate();
             ExcelWorksheet sheet = package.Workbook.Worksheets["result"];
             ExcelRangeBase exportRange = sheet.Cells["A1:E24"];
             IExcelHtmlRangeExporter exporter = exportRange.CreateHtmlExporter();
             string Html = exporter.GetSinglePage();
             File.WriteAllText(@"ceil-vlookup.html", Html);
     }
 }
```

**Comment by colbybhearn** [comment]
I was advised to try Ceiling.Math as well, but the overall result was still `#Value!`

**Comment by JanKallman** [comment]
Ok, I'll add a fix for `Ceiling.Math` as well.

**Comment by JanKallman** [comment]
Fixed in 6.2.9
